docs/eigenda/operator-guides/run-a-node/registration.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+---
+
+# Register Your Operator
+
+Your operator will not begin receiving traffic from the EigenDA disperser until it has registered for one or more quorums with EigenDA. Note, as discussed in [delegation requirements](../requirements/delegation-requirements/) that registration with an EigenDA quorum requires that an operator already be [registered as an operator with EigenLayer](https://docs.eigenlayer.xyz/operator-guides/operator-installation) and to have a minimum amount of stake delegated within each quorum to be registered. 
```

Comment:
I think there's an ordering issue. EigenLayer Registration should happen before you run the docker. Reason being - During EigenLayer registration you ask them to create BLS and ECDSA keys. BLS keys is required to run docker. so maybe there should be section for that?

Reply:
I'll add a section in the `run-with-docker` page calling this out. I think we shouldn't explicitly include the steps for doing EigenLayer registration in our docs, since this is out of scope, but it makes sense to have callouts and references at appropriate locations.

Reply:
oh ya that makes sense. Just add a pre req sort of thing.